bundles/org.openhab.binding.insteon2/README.md

```diff
@@ -0,0 +1,941 @@
+# Insteon Binding
```

Comment:
```suggestion
# Insteon2 Binding
```

Reply:
"This is why I am submitting a new separate binding with the expectation that the current one would be renamed as legacy allowing existing users to keep using that binding until they decide to move to the new one."

Reply:
@robnielsen Is there a reason why you think your binding shouldn't be renamed as legacy when this binding gets merged?

Reply:
@jsetton it should not be renamed legacy. Nobody agreed to rename the existing binding to legacy.